tools/update_nfx_docker_compose.sh

```diff
@@ -19,13 +19,13 @@ if [ -z "$NFI_PATH" ]; then
     exit 1
 fi
 
+echo "starting update on path $NFI_PATH"
 cd $NFI_PATH
 
 export $(grep -v '^#' .env | xargs)
```

Comment:
Since the path of .env location is not configurable, it's not crystal clear that the script must be moved to repo root. 
But if you just concatenate it to the repo path variable, it's a bit more obvious.

Reply:
An helper function like that one:  

load_env() {
    local env_file="${1:-.env}"
    
    if [[ -f "$env_file" ]]; then
        set -a
        source <(grep -v '^#' "$env_file" | grep -v '^[[:space:]]*$' | sed 's/^/export /')
        set +a
    else
        echo "$env_file not found"
        exit 1
    fi
}

load_env "${var_path}/.env"